src/ray/common/id.cc

```diff
@@ -26,6 +26,14 @@ std::mt19937 RandomlySeededMersenneTwister() {
 
 uint64_t MurmurHash64A(const void *key, int len, unsigned int seed);
 
+WorkerID ComputeDriverIdFromJob(const JobID &job_id) {
+  std::vector<uint8_t> data(WorkerID::Size(), 0);
```

Comment:
This could potentially be done better by defining WorkerID like this:
```
class WorkerID {
  //...
private:
  uint32_t job_id_;
  uint8_t extra_data_[16];
}
```

Reply:
Yes. But I'd like to do this is another separated PR which aims to separate `WorkerID` from `UniqueID` and distinguish `WorkerID` from `ClientID`.

Reply:
Then please add a TODO explaining this.

Reply:
There is a TODO in `id.h` already.